test/e2e/mattermost_test.go

```diff
@@ -0,0 +1,135 @@
+package e2e
+
+import (
+	goctx "context"
+	"fmt"
+	"testing"
+	"time"
+
+	apis "github.com/mattermost/mattermost-operator/pkg/apis"
+	operator "github.com/mattermost/mattermost-operator/pkg/apis/mattermost/v1alpha1"
+	mysqlOperator "github.com/oracle/mysql-operator/pkg/apis/mysql/v1alpha1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 300
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+)
+
+func TestMattermost(t *testing.T) {
+
+	mysqlList := &mysqlOperator.ClusterList{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Cluster",
+			APIVersion: "mysql.oracle.com/v1alpha1",
+		},
+	}
+	err := framework.AddToFrameworkScheme(mysqlOperator.AddToScheme, mysqlList)
+	if err != nil {
+		t.Fatalf("failed to add mysql custom resource scheme to framework: %v", err)
```

Comment:
Can we use the testify packages like we do in any new tests in the server? They're much cleaner and easier to work with

* https://godoc.org/github.com/stretchr/testify/require
* https://godoc.org/github.com/stretchr/testify/assert

Reply:
sure! will make the changes

Reply:
done @jwilander